Dockerfile

```
@@ -88,7 +88,16 @@ ENV NEXT_PUBLIC_WEBAPP_URL=$NEXT_PUBLIC_WEBAPP_URL \
 ENV NODE_ENV=production
 EXPOSE 3000
 
+# Secrets Manager hydration (scripts/hydrate-env.*). Installed to /opt/hydrate
+# rather than the repo's dependency graph on purpose: this is deployment
+# plumbing, not an app dependency, and keeping it out of package.json means it
+# cannot collide when merging upstream.
+COPY scripts/hydrate-env.js scripts/hydrate-env.sh /opt/hydrate/
+RUN chmod +x /opt/hydrate/hydrate-env.sh \
```

Hadolint (2.15.1)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)